backend/app/views/spree/admin/shared/_calculator_fields.html.erb

```diff
@@ -4,6 +4,7 @@
   <div id="preference-settings">
     <div class="field">
       <%= f.label(:calculator_type, Spree::Calculator.model_name.human) %>
+      <%= admin_hint t("spree.#{@calculator_type}"), t(@calculator_type, scope: [:spree, :hints, "spree/calculator"]) %>
```

Comment:
I couldn't find a great way to grab the parent module of the calculator without it getting hacky so I figured an instance method in the controller would be a decent alternative!